{
  "message": "Internal R error",
  "r_error_message": "Error in validate_data(target, data_period, question_period) : \n  Historical data in request appear to not be indexed with correct dates. The question period starts 2018-04-22 and dates like [..., 2016-08-15, 2016-12-16, 2017-04-18, 2017-08-19, 2017-12-20, 2018-04-22] are expected in the historical data, but instead they have [..., 2015-12-12, 2016-04-13, 2016-08-14, 2016-12-15, 2017-04-17, 2017-08-18].\nCalls: r_basil_ts -> validate_data\nExecution halted\n"
}